dmithamo/mydiary
# #159101719 Build out PUT /entries/id Endpoint

#### What does this PR do?
This PR builds out the endpoint defined in the title above.

#### Description of Task to be completed?
Requests to `PUT mydiary/api/v1/entries/id` can now be serviced.
#### How should this be manually tested?
See [README](https://github.com/dmithamo/mydiary/tree/ft-api-modify-entry-159101719) on how to set up the server and test the endpoint.
#### What are the relevant pivotal tracker stories?
[#159101719](https://www.pivotaltracker.com/story/show/159101719)
